scripts/build.mjs

```diff
@@ -11,7 +11,9 @@ export default {
     { file: './dist/index.js', format: 'cjs' },
   ],
   plugins: [
-    typescript(),
+    typescript({
+      exclude: ['test/**'],
```

Comment:
is this still needed with the workspaces?

Reply:
Right, not anymore. removed it